test/optests_failures_dict.json

```diff
@@ -1,5 +1,16 @@
 {
   "_description": "This is a dict containing failures for tests autogenerated by generate_opcheck_tests. For more details, please see https://docs.google.com/document/d/1Pj5HRZvdOq3xpFpbEjUZp2hBovhy7Wnxw14m6lF2154/edit",
   "_version": 1,
-  "data": {}
+  "data": {
+    "torchvision::roi_align": {
+      "TestRoIAlign.test_aot_dispatch_dynamic__test_mps_error_inputs": {
+        "comment": "RuntimeError: MPS does not support roi_align backward with float16 inputs",
+        "status": "xfail"
+      },
+      "TestRoIAlign.test_autograd_registration__test_mps_error_inputs": {
+        "comment": "NotImplementedError: autograd_registration_check: NYI devices other than CPU/CUDA, got {'mps'}",
+        "status": "xfail"
+      }
+    }
+  }
```

Comment:
I'm leaving the mps-related stuff as xfail for now, we can fix them eventually in other PRs.